From: Ben Hutchings <ben@decadent.org.uk>
Date: Thu, 7 Jun 2012 04:40:34 +0100
Subject: [PATCH] IA64: Export <asm/cmpxchg.h> to userland

Parts of <asm/intrinsics.h> were moved to <asm/cmpxchg.h>, and the
former now includes the latter.  The former is exported to userland
so the latter must be too.

Cc: stable@vger.kernel.org # v3.4
Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 arch/ia64/include/asm/Kbuild |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/ia64/include/asm/Kbuild b/arch/ia64/include/asm/Kbuild
index 241d1c5..c929a53 100644
--- a/arch/ia64/include/asm/Kbuild
+++ b/arch/ia64/include/asm/Kbuild
@@ -6,6 +6,7 @@ header-y += gcc_intrin.h
 header-y += ia64regs.h
 header-y += intel_intrin.h
 header-y += intrinsics.h
+header-y += cmpxchg.h
 header-y += perfmon.h
 header-y += perfmon_default_smpl.h
 header-y += ptrace_offsets.h
-- 
1.7.10

